docs/transparent-validation/rte/v1.1.0/flashing-guide.md

```diff
@@ -14,6 +14,10 @@ RTE.
 
 ### Flashing
 
+> Note: the `flash.sh` script, used in this chapter, is available only in 0.8.x
```

Comment:
Please go through this document in more details. We instruct to download flash.sh from github and scp to RTE. This is not needed on newer versions of RTE OS. May still be required on older.

Reply:
Thanks, fixed [here](https://github.com/Dasharo/docs/compare/768792607d870545885d08ea25768b19133565eb..c72463f455aaa77c59700011bb5d83c744407dc2)